step-api/src/test/java/org/jenkinsci/plugins/workflow/steps/StepConfigTester.java

```diff
@@ -57,7 +58,8 @@ public StepConfigTester(JenkinsRule rule) {
         FreeStyleProject p = rule.createFreeStyleProject();
         try {
             p.getBuildersList().add(new StepBuilder(before));
-            p = rule.configRoundtrip(p);
+            rule.submit(rule.createWebClient().getPage(p,"configure").getFormByName("config"));
+            //p = rule.configRoundtrip((Project<FreeStyleProject,FreeStyleBuild>)p);
```

Comment:
Try creating a method in _this_ class like

```java
private static <P extends Job> P configRoundtrip(JenkinsRule rule, P job) throws Exception {
    rule.configRoundtrip(job);
}
```

and then change the above code to

```java
p = configRoundtrip(rule, p);
```